hillel_django/schema.py

```diff
@@ -88,6 +88,44 @@ def mutate(cls, root, info, name, price, description=None):
         return ProductMutation(product=product)
 
 
+class OrderInput(graphene.InputObjectType):
+    products = graphene.List(graphene.ID, required=True)
+
+
+class OrderType(DjangoObjectType):
+    class Meta:
+        model = Order
+        fields = (
+            'id',
+            'user',
+            'products',
+            'total_price',
+            'bill',
+        )
+
+
+class CreateOrder(graphene.Mutation):
+    class Arguments:
+        order_input = OrderInput(required=True)
+
+    order = graphene.Field(OrderType)
+
+    @staticmethod
+    def mutate(root, info, order_input):
+        if not order_input.products:
+            raise Exception('An order must contain at least one product.')
+
+        order = Order.objects.create()
+
+        for product_id in order_input.products:
+            product = Product.objects.get(id=product_id)
```

Comment:
pls add validation, that the product has a valid id.

Reply:
so if you do an order mutation but you pass the product which does not exist, here you will receive an exception. So can you handle this exception, so you will see correct response in GraphQL.